apps/DesktopStreamer/main.cpp

```diff
@@ -40,14 +40,16 @@
 
 #include "MainWindow.h"
 
+#include <deflect/utils.h>
 #include <deflect/version.h>
 
 #include <QCommandLineParser>
 
 int main(int argc, char* argv[])
 {
-    QApplication app(argc, argv);
+    deflect::blockFalseWarnings();
```

Comment:
The application should not see this. Have a static initializer in the method which emits this warning?